library/src/main/kotlin/com/connectrpc/http/HTTPClientInterface.kt

```diff
@@ -46,92 +45,3 @@ interface HTTPClientInterface {
      */
     fun stream(request: HTTPRequest, duplex: Boolean, onResult: suspend (StreamResult<Buffer>) -> Unit): Stream
 }
-
-interface Stream {
```

Comment:
Minor cleanup in here: I hoisted the `Stream` stuff out to a separate `Stream.kt` file.